arbisoft/arbisoft/spiders/jobs_spider.py

```diff
@@ -0,0 +1,55 @@
+import scrapy
+import datetime
+
+
+class JobsSpider(scrapy.Spider):
+    name = 'jobs'
+    start_urls = [
+        'https://news.ycombinator.com/jobs'
+    ]
+
+    def parse(self, response):
+
+        items = response.css("tr.athing, td.subtext")
+        all_job_titles = []
```

Comment:
No need for `all` in `all_job_titles`. `job_titles` will do just fine.

Reply:
job_titles is another variable name ...I'll have to change that as well

Reply:
ok resolved...changed all_job_titles to job_titles